generator/internal/sidekick/sidekick_test.go

```diff
@@ -25,17 +25,15 @@ import (
 	"testing"
 )
 
-func TestMain(m *testing.M) {
-	flag.Parse()
-	os.Exit(m.Run())
-}
+const (
+	// projectRoot is the root of the google-cloud-rust. The golden files for
+	// these tests generate code in .,./../auth and ../../src/gax
```

Comment:
I misspoke, they *depend* on code from ``../../auth`. and `../../src/gax`, they do not affect that code.

nit: extra comma in the `auth` path.